packages/cli/package.json

```diff
@@ -3,9 +3,7 @@
   "description": "CLI to interact with Heroku",
   "version": "7.67.1",
   "author": "Jeff Dickey @jdxcode",
-  "bin": {
-    "heroku": "./bin/run"
-  },
+  "bin": "./bin/run",
```

Comment:
I'm not sure why yarn 3 requires this format for generating a bin file, but it uses the package name, which is also "heroku", and it works the saem.

Reply:
That's interesting... It looks like the `{}` format should work from the [Yarn documentation](https://yarnpkg.com/configuration/manifest#bin)...

Reply:
Does the `build` command work?

Reply:
It does and the bin file is generated correctly.
<img width="344" alt="image" src="https://user-images.githubusercontent.com/6306282/214391617-893b1336-5bcf-4a73-bca8-2f835d62752b.png">

Reply:
😌 Thank goodness.